src/html-entrypoint.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const { JSDOM } = require('jsdom');
+const fs = require('fs');
+const path = require('path');
+
+function htmlEntrypoint(appName, distPath, htmlPath) {
+  let html = fs.readFileSync(path.join(distPath, htmlPath), 'utf8');
+  let dom = new JSDOM(html);
+  let scripts = [];
+
+  let config = {};
+  for (let element of dom.window.document.querySelectorAll('meta')) {
+    let name = element.getAttribute('name');
+    if (name && name.endsWith('/config/environment')) {
+      let content = JSON.parse(decodeURIComponent(element.getAttribute('content')));
+      content.APP = Object.assign({ autoboot: false }, content.APP);
+      config[name.slice(0, -1 * '/config/environment'.length)] = content;
```

Comment:
Is there a way to avoid this? Seems a bit hacky IMO

Reply:
Would you prefer `name.split('/')[0]`?

Reply:
yes that is a lot clearer :) Thanks!